BUILD.bazel

```diff
@@ -36,6 +36,7 @@ goimports(
     exclude_paths = [
         "./vendor/*",
         "./.history/*",
+        "./.git/*",
```

Comment:
Why this change is needed?

Reply:
It's not suppose to be here, will remove it.

The story about this is here, https://github.com/kubevirt/kubevirt/pull/4330